components/ConversationsList.tsx

```diff
@@ -143,20 +143,44 @@ const ConversationsList = ({
 
   return (
     <div>
-      {conversations &&
-        conversations.sort(orderByLatestMessage).map((convo) => {
-          const isSelected =
-            router.query.recipientWalletAddr == convo.peerAddress
-          return (
-            <ConversationTile
-              key={convo.peerAddress}
-              conversation={convo}
-              isSelected={isSelected}
-            />
-          )
-        })}
+      {conversations && conversations.size > 0 ? (
+        Array.from(conversations.values())
+          .sort(orderByLatestMessage)
+          .map((convo) => {
+            const isSelected =
+              router.query.recipientWalletAddr == convo.peerAddress
+            return (
+              <ConversationTile
+                key={convo.peerAddress}
+                conversation={convo}
+                isSelected={isSelected}
+              />
+            )
+          })
+      ) : (
+        <NoConversationsMessage />
+      )}
+    </div>
+  )
+}
+
+const NoConversationsMessage = (): JSX.Element => {
+  return (
+    <div className="flex flex-col flex-grow justify-center">
+      <div className="flex flex-col items-center px-4 text-center">
+        <ChatIcon
+          className="h-8 w-8 mb-1 stroke-n-200 md:stroke-n-300"
+          aria-hidden="true"
+        />
+        <p className="text-xl md:text-lg text-n-200 md:text-n-300 font-bold">
+          Your message list is empty
+        </p>
+        <p className="text-lx md:text-md text-n-200 font-normal">
+          There are no messages in this wallet
```

Comment:
@saulmc This is new copy. WDYT the right message should be for this empty state

Reply:
Nvm. This was copied over from another file. Although I think our usage of the term "wallet" here doesn't line up with our style guide. But we can fix that in a subsequent PR

Reply:
Here is a quick suggestion to unblock the PR:

```suggestion
          There are no messages for this address
```

cc @darickdang, @jhaaaa

Reply:
Got it